pep/app/pods/components/page/drawer/component.ts

```diff
@@ -19,13 +21,18 @@ export default class PageDrawer extends Component<PageDrawerArgs> {
     @service auth!: AuthService;
     @service modal!: ModalService;
     @service configuration!: ConfigurationService;
+    @service currentUser!: CurrentUserService;
 
     @tracked isUserMenuOpen = false;
 
     get defaultSearchParams() {
         return this.configuration.defaultSearchParams;
     }
 
+    get readDisabled() {
+        return !!!this.currentUser.lastViewedDocumentId;
```

Comment:
@bakerac4 missed this, but we should fix this on the next branch since its unnecessary and the page/nav/component's version correctly only does `return !this.currentUser.lastViewedDocumentId;`